modules/microshift-adding-service-to-blueprint.adoc

```diff
@@ -9,11 +9,19 @@
 
 Adding the {microshift-short} RPM package to an Image Builder blueprint enables the build of a {op-system-ostree} image with {microshift-short} embedded.
 
+* If your goal is a faster installation, use step 1 and create your own minimal blueprint file.
+* If your goal is a faster startup for {microshift-short}, use the generated blueprint in step 2. Using the generated blueprints is a longer installation process, but a faster start up because container references are accessed locally.
++
+[IMPORTANT]
+====
+Replace <minimal-microshift> in the following procedures with the name of the TOML you are using.
+====
```

Comment:
Can we remove this please and use a file name in the commands rather than `<some name>`?
The latter really annoys when copy-pasting commands for execution.

Reply:
We have references to this `minimal-microshift` in a few places. I'd suggest to replace it by `microshift-blueprint` to be less ambiguous.